### PR TITLE
Add DLL version to rlottie.dll

### PR DESCRIPTION
**Pull request by dchengTSC**

### CODE REVIEW

**Review by KMojek — APPROVED** [review]
👍

File: vs2019/rlottie.rc [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", "TechSmith Corporation"
+            VALUE "FileDescription", "rlottie"
+            VALUE "FileVersion", "1.0.0.1"
+            VALUE "InternalName", "rlottie.dll"
+            VALUE "LegalCopyright", "Copyright � 2020 TechSmith Corporation. All rights reserved."
```
Comment (KMojek) [inline vs2019/rlottie.rc]: Actually, is this correct for third-party code? Pango credits Red Hat Software. MP3lame attributes someone else; looks like mp4v2 doesn't mention a company.

Comment (qunliu) [reply vs2019/rlottie.rc]: FYI, we include this version header file, take CSRenderLib.rc as example: https://github.com/TechSmith/Luma/blob/master/CSRenderLib/CSRenderLib.rc#L53

This is how we may use it: https://github.com/TechSmith/Luma/blob/master/CSRenderLib/CSRenderLib.rc#L72

Comment (qunliu) [reply vs2019/rlottie.rc]: Oh, never mind. This is not part of our internal code. What you do is probably okay, just make sure we bump the version each time we have changes.

Comment (dchengTSC) [reply vs2019/rlottie.rc]: Maybe I'll just leave the company name off

**Pull request merged**